components/wb-bubble/wb-bubble-va-en.html

```diff
@@ -0,0 +1,59 @@
+---
+title: Bubble - Open Overlay
+language: en
+category: Plugins
+description: Working example of the virtual assistant bubble with customized avatar.
+altLangPage: wb-bubble-fr
+dateModified: 2021-06-21
+---
+<span class="wb-prettify all-pre hide"></span>
+
+<h2>Virtual assistant bubble including custumized avatar</h2>
```

Comment:
H2 isn't necessary in my opinion, since its the only heading it could just be appended to the H1 instead, so the H1 could be: `Bubble - open overlay for virtual assistant including customized avatar`